docs/serving/custom-model-serving-class.md

```diff
@@ -167,8 +167,9 @@ You can also deploy a model from within an ML pipeline (check the various demos
 ## Model monitoring
 
 Model activities can be tracked into a real-time stream and time-series DB. The monitoring data
-is used to create real-time dashboards and track model accuracy and drift. 
-To set the tracking stream options, specify the following function spec attributes:
+is used to create real-time dashboards, track model accuracy, detect drift, and analyze performance. 
+
+To monitor a deployed model, apply `set_tracking()` on your serving function and specify the function spec attributes:
 
         fn.set_tracking(stream_path, batch, sample)
```

Comment:
same

```suggestion
        fn.set_tracking(stream_path, batch, sample, tracking_policy)
```

+ add the following description:
`tracking_policy` - optional, model tracking configurations, such as setting the scheduling policy of the model monitoring batch job.

Reply:
added description